src/components/Input.tsx

```diff
@@ -40,6 +57,7 @@ export const Input = React.forwardRef<TextInput, InputProps>((props, ref) => {
     loadingPosition = 'right',
     iconRight,
     iconLeft,
+    labelStyle = {},
```

Comment:
there is no need for default object, it looks weird to me

Reply:
okay